src/entrypoint/controllers/quey.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { NextFunction, Request, Response } from 'express';
```

Comment:
There is a typo in the name of this file

Reply:
Renamed file.